test-config.edn

```diff
@@ -10,5 +10,5 @@
  :authentication :fake
  :languages [:en :fi :sv]
  :enable-pdf-api true
- :catalogue-is-public true}
+ :catalogue-is-public false}
```

Comment:
I thought we'd keep it `true` in dev and test. I don't mind either way, but repeating the default is a bit funny.

Reply:
I think it's perhaps not the direction we are going to so I made it false. I could remove those two but it's slightly easier to turn it on now.